src/main/views/delete-court-publication-success.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "template.njk" %}
+{% from "./macros/common-components.njk" import successPanel %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
+  {{ successPanel(panelTitle, panelBodyMessage) }}
+  <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-top-6">{{ nextMessage }}</p>
+  <p class="govuk-body govuk-!-margin-0 govuk-!-font-weight-bold"><a class="govuk-link" href="delete-court-reference-data">{{ removeCourt }}</a></p>
```

Comment:
As discussed separately, this should be re-worded / re-direct the user back to the delete court page for the specific court

Reply:
new link added.